stable/kasm/1.12.0/README.md

```diff
@@ -0,0 +1,37 @@
+# Introduction
+
+Kasm Workspaces streaming containerized apps and desktops to end-users.
+
+TrueCharts are designed to be installed as TrueNAS SCALE app only. We can not guarantee this charts works as a stand-alone helm installation.
+**This chart is not maintained by the upstream project and any issues with the chart should be raised [here](https://github.com/truecharts/apps/issues/new/choose)**
+
+## Source Code
+
+* <https://github.com/thelamer/charts-test/tree/main>
+
+## Requirements
+
+Kubernetes: `>=1.16.0-0`
+
+## Dependencies
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://truecharts.org | common | 8.5.4 |
+
+## Installing the Chart
+
+To install this App on TrueNAS SCALE check our [Quick-Start Guide](https://truecharts.org/manual/Quick-Start%20Guides/03-Installing-an-App/).
+
+## Uninstalling the Chart
+
+To remove this App from TrueNAS SCALE check our [Quick-Start Guide](https://truecharts.org/manual/Quick-Start%20Guides/07-Deleting-an-App/).
+
+## Support
```

Comment:
forgive my ignorance on what is common for Truenas projects (or anything you and Justin discussed on this), but all these support options seem to point to truenas specific support, should any of these point to our public issue tracker on github?

Reply:
This readme is not presented to the user it is a git thing and was copied over from truecharts for the most part including the library for compatibility.  Though the app-readme is, I have pulled out those references as they are not relevant outside of using their library.